server/src/main/kotlin/io/provenance/objectstore/gateway/configuration/DataConfig.kt

```diff
@@ -52,31 +52,45 @@ class DataConfig {
             addDataSourceProperty("protocol.io.threads", "12")
         }.let { HikariDataSource(it) }
     }
-}
-
-@Component
-class DataMigration(dataSource: DataSource, databaseProperties: DatabaseProperties) {
-    companion object : KLogging()
 
-    init {
-        Database.connect(dataSource)
-        Database.registerDialect("pgsql") { PostgreSQLDialect() }
-        Database.registerDialect("sqlite") { SQLiteDialect() }
-        transaction {
-            if (databaseProperties.type == "postgresql") {
-                try {
-                    SchemaUtils.createSchema(Schema(databaseProperties.schema))
-                } catch (e: Exception) {
-                    logger.info("Exception creating schema [${databaseProperties.schema}]: ${e.message}")
-                    logger.info("attempting to continue as this may be just a permissions issue (the schema may already exist or you might need to create it manually)")
+    @Bean("databaseConnect")
+    fun databaseConnect(dataSource: DataSource, databaseProperties: DatabaseProperties): Database = Database.connect(dataSource)
+        .also {
+            TransactionManager.manager.defaultIsolationLevel = Connection.TRANSACTION_READ_COMMITTED
+            Database.registerDialect("pgsql") { PostgreSQLDialect() }
+            Database.registerDialect("sqlite") { SQLiteDialect() }
+            transaction {
+                if (databaseProperties.type == "postgresql") {
+                    try {
+                        SchemaUtils.createSchema(Schema(databaseProperties.schema))
+                    } catch (e: Exception) {
+                        logger.info("Exception creating schema [${databaseProperties.schema}]: ${e.message}")
```

Comment:
Nit: Should these be warns?  Or do they show up every time the app runs when the schema already exists?  If they always show up, infos are cool

Reply:
good call, I'll make them warns